packages/worker/src/worker.ts

```diff
@@ -307,12 +316,14 @@ export class Worker {
   protected constructor(
     protected readonly nativeWorker: NativeWorkerLike,
     /**
-     * Optional WorkflowCreator - if not provided, Worker will not poll on workflows
+     * Optional WorkflowCreator - if not provided, Worker will not poll on Workflows
      */
     protected readonly workflowCreator: WorkflowCreator | undefined,
-    public readonly options: CompiledWorkerOptions
+    public readonly options: CompiledWorkerOptions,
+    protected readonly dataConverter: LoadedDataConverter
```

Comment:
I liked that we had a "usable" data converter in the public `CompiledWorkerOptions`, would be nice to get that back